apps/editors/helpers.py

```diff
@@ -476,8 +478,13 @@ def get_actions(self, request, addon):
         labels, details = self._review_actions()
 
         actions = SortedDict()
-        if not addon.admin_review or acl.action_allowed(
-                request, 'ReviewerAdminTools', 'View'):
+        if ((not addon.admin_review or acl.action_allowed(request,
+             'ReviewerAdminTools', 'View'))
+            and (not GroupUser.objects.filter(group=Group.objects.get(
+                 name='Limited Reviewers'), user=request.user)
```

Comment:
that's fine, you can just import it like `from apps.editors.views import is_limited_reviewer`.